### ISSUE TITLE
els_VIDTrigMvaValue empty 2016 94x miniaodv3 cms4

### ISSUE THREAD

**Issue by aminnj** [cmstas/NtupleMaker#30]
E.g., /hadoop/cms/store/group/snt/run2_data2016_94x//DoubleMuon_Run2016F-17Jul2018-v1_MINIAOD_CMS4_V10-02-04/merged_ntuple_1.root
has empty `els_VIDTrigMvaValue` branch

MC seems fine

**Comment by gzevi** [comment]
I don't think we ever used the "Triggered MVA", and it doesn't exist after 2016, so I think this is fine, no? Does the untriggered MVA look OK?

**Comment by aminnj** [comment]
I didn't flesh out the original error.
`els_VIDTrigMvaValue` and `els_VIDNonTrigMvaValue` are empty
but `els_VIDSpring16GPMvaValue` looks fine
